gui/wxpython/datacatalog/infomanager.py

```diff
@@ -57,5 +57,18 @@ def ShowImportDataInfo(self, OnImportOgrLayersHandler, OnImportGdalLayersHandler
         ).format(loc=gisenv()['LOCATION_NAME'])
         self.infoBar.ShowMessage(message, wx.ICON_INFORMATION, buttons)
 
+    def ShowImportSuccessfulInfo(self):
+        """Show info about the Display and Modules tabs focused on the first-time user"""
+        buttons = [("See documentation", self._onSeeDocumentation)]
+        message = _(
+            "You have successfully imported data. To change layer properties, "
+            "go to Display tab below and click over layer name.\n\n"
```

Comment:
either 'double-click on a layer' or 'right-click on a layer'

Reply:
I have changed it to 'right-click on a layer name'